articles/153_roslaunch_remote_launch.md

```diff
@@ -0,0 +1,185 @@
+# Multi-Machine and Remote Launching
+## Goals
+- Connect to a remote host and run nodes on it
+    - Ensure this capability does not compromise security
```

Comment:
We're currently working on launch integration for security. In order to make sure it scales to remote launching, we might want to consider ways to send security files over to the remote machine.